basis/cpu/architecture/architecture.factor

```diff
@@ -234,7 +234,7 @@ HOOK: %replace-imm cpu ( src loc -- )
 HOOK: %inc cpu ( loc -- )
 
 HOOK: stack-frame-size cpu ( stack-frame -- n )
-HOOK: %call cpu ( word -- )
+HOOK: %call cpu ( word height -- )
```

Comment:
Does any implementation of ``#call`` use the height argument? It looks like they always pass ``0`` and always ``drop`` the height?